libs/damap/src/lib/components/dmp/licenses/licenses.component.html

```diff
@@ -1,64 +1,73 @@
             </div>
             <ng-container *ngIf="dataset.value.personalData || dataset.value.sensitiveData">
-              {{'dmp.steps.licensing.warning.personalData' | translate}}
+              {{ 'dmp.steps.licensing.warning.personalData' | translate }}
               <ng-container *ngIf="!isAnonymisedOrPseudonymised">
-                {{'dmp.steps.licensing.warning.pseudonymisation' | translate}}
+                {{ 'dmp.steps.licensing.warning.pseudonymisation' | translate }}
               </ng-container>
             </ng-container>
             <ng-container *ngIf="dataset.value.legalRestrictions">
-              {{'dmp.steps.licensing.warning.legalRestrictions' | translate}}
+              {{ 'dmp.steps.licensing.warning.legalRestrictions' | translate }}
             </ng-container>
           </div>
-          <div class="mat-card-container">
+          <div id="licensingContainer" class="mat-card-container">
             <mat-form-field class="full-width">
-              <mat-label>{{'dmp.steps.licensing.question.license' | translate}}</mat-label>
-              <mat-select formControlName="license">
+              <mat-label>{{
+                'dmp.steps.licensing.question.license' | translate
+                }}</mat-label>
+              <mat-select [(value)]="defaultLicense">
                 <mat-option *ngFor="let license of licenses" [value]="license.url | translate">
-                  {{license.name | translate}}
+                  {{ license.name | translate }}
                 </mat-option>
               </mat-select>
             </mat-form-field>
             <app-license-wizard (selectedLicense)="setLicenseSelectorResult($event, i)"></app-license-wizard>
           </div>
-          <div class="mat-card-container">
+          <div id="licensingContainer" class="mat-card-container">
             <mat-form-field class="date-form">
-              <mat-label>{{'dmp.steps.licensing.question.startDate' | translate}}</mat-label>
-              <input matInput [matDatepicker]="picker_i" formControlName="startDate">
+              <mat-label>{{
+                'dmp.steps.licensing.question.startDate' | translate
+                }}</mat-label>
+              <input matInput [matDatepicker]="picker_i" formControlName="startDate" />
               <mat-datepicker-toggle matSuffix [for]="picker_i"></mat-datepicker-toggle>
               <mat-datepicker #picker_i></mat-datepicker>
-              <mat-hint
-                *ngIf="dmpForm.controls.project?.value?.end === dataset.value.startDate">
-                {{'dmp.steps.licensing.question.hint' | translate}}
+              <mat-hint *ngIf="
+                  dmpForm.controls.project?.value?.end === dataset.value.startDate
+                ">
+                {{ 'dmp.steps.licensing.question.hint' | translate }}
               </mat-hint>
             </mat-form-field>
           </div>
+          <div id="accessTypeContainer" class="mat-card-container">
```

Comment:
`dataAccess` disappears once it's been changed to something other than `OPEN`. Keep it visible, so that it may be changed again.